test/endpoints/baz/baz_simpleservice_method_compare_test.go

```diff
@@ -196,5 +196,5 @@ func TestCompareInvalidArgs(t *testing.T) {
 	)
 
 	assert.Nil(t, err)
-	assert.Equal(t, 400, res.StatusCode)
+	assert.Equal(t, 500, res.StatusCode)
```

Comment:
The request made here has a missing required field: bytes.NewBuffer([]byte(`{
			"arg2":{ "b1":true,"s2":"a","i3":1 }
		}`)), 

This was previously caught by easyjson to send a 400 Bad Request.  In encoding/json this will be 500,  unless there is a validation framework similar to easysjon. We don't plan to have  field level validation in Zanzibar at this time